hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketInfo.java

```diff
@@ -96,6 +96,10 @@ public final class OmBucketInfo extends WithObjectID implements Auditable, CopyO
   private long usedNamespace;
   private final long quotaInBytes;
   private final long quotaInNamespace;
+  // Total size of data trapped which is pending to be deleted either because of data trapped in snapshots or
+  // background key deleting service is yet to run.
```

Comment:
```suggestion
  // Total size of data trapped which is pending to be deleted either because of data trapped in snapshots or
  // background key deleting service is yet to run.
  // This is also the size exclusively held by the snapshot.
  // i.e. when all snapshots of this bucket are deleted and purged, this much space would be released.
```

Reply:
done